core/domain/question_services.py

```diff
@@ -585,21 +585,6 @@ def get_question_summaries_by_creator_id(creator_id):
     return question_summaries
 
 
-def get_question_summary_by_question_id(question_id, strict=False):
```

Comment:
@aks681 this function is not used anywhere that's why I removed it. But the e2e tests are failing and I don't know whether its due to the removal of this. Can you check? Thanks!

Reply:
Hmm, I don't think this is the reason, as the function is not called anymore. Also, it is not the question editor e2e test that is failing, right?

Reply:
The tests passed. PTAL!